vmware/ngc/NGC-Register/README.md

```diff
@@ -0,0 +1,6 @@
+###NGC Plugin Register Guide
+### support https/http
```

Comment:
1) Please add a brief introduction to Registration
2) Dependency on other PRs or Modules as needed.
3) Can put the overall view of modules and positioning of registration 
(Can update after the merge as a fix.

Reply:
NGC Plugin Register Guide updated.